src/meetings/2021-11-22.md

```diff
@@ -0,0 +1,31 @@
+# 2021-11-22
+
+| Info                                           |                          |
+| ---------------------------------------------- | ------------------------ |
+| Time                                           | [See in your time zone]  |
+| Calendar event (with Zoom details)             | [link][cal]              |
+| Social hour calendar event (with Zoom details) | [link][calsh]            |
+| Recording                                      | (link posted after call) |
+
+[see in your time zone]: https://everytimezone.com/s/91c9791f
+[cal]: https://calendar.google.com/event?action=TEMPLATE&tmeid=NnVibTVzcmEwdnZxMTBtYjZ0ZmZxaWtyYXQgN24wdnZvcWZlMGtibms2aTA0dWl1NTJ0MzBAZw&tmsrc=7n0vvoqfe0kbnk6i04uiu52t30%40group.calendar.google.com
+[calsh]: https://calendar.google.com/event?action=TEMPLATE&tmeid=NGd2N2FzYjBnMHFiNXA4dWdtYWZ2ZmxobmwgN24wdnZvcWZlMGtibms2aTA0dWl1NTJ0MzBAZw&tmsrc=7n0vvoqfe0kbnk6i04uiu52t30%40group.calendar.google.com
```

Comment:
Previously, our calendar links have just been links that allow people to add the event to their calendar, but I don't think it adds them as "attending". This is the link I got from "publishing" the event, which should mean that if someone comes out, they'll be notified next time the event is duplicated. Let me know if this isn't a good way to do it.

Reply:
I think I always created the links by "publishing", but tbh I don't really understand how google calendar works.